### ISSUE TITLE
IS there Array size support?

### ISSUE THREAD

**Issue by BezBartek** [piccolo-orm/piccolo#781]
Am I able to define Array size? I do not see it in docs anywhere, as well as in column type definition
If that feature is absent, is it possible to add it in near future?
Thanks!

**Comment by dantownsend** [comment]
Yeah, it's not possible to specify the array size currently, but it would be a very simple change.

However, looking at the [Postgres docs](https://www.postgresql.org/docs/current/arrays.html#ARRAYS-DECLARATION) it doesn't seem to do anything with the array size.

<img width="705" alt="Screenshot 2023-03-03 at 12 17 08" src="https://user-images.githubusercontent.com/350976/222718354-98e78669-22c3-4ac4-8523-69c033fbefa7.png">

Does specifying the array size give any benefits besides documenting the intended size?

**Comment by dantownsend** [comment]
I guess we could do some validation on Piccolo's side when someone passes in an array to make sure it's the required size.

**Comment by BezBartek** [comment]
@dantownsend 
Ok, I didn't know that it does nothing on postgres level, I just compared it to Django, where I used to define the size of arrays. 
In my case, it would be hopeful cus it have to be always array of size 3, and now I will have to add validation of that myself 

I think it would be nice to define it on the model level, cus configuration like that from a business perspective is part of the model schema

**Comment by dantownsend** [comment]
Yeah, interesting idea. We could add validation to the Pydantic model too based on the array size.
